markdown/api.md

```diff
@@ -362,19 +362,24 @@ forceResize()
 useFrame((callback: (state, delta) => void), (renderPriority: number = 0))
 ```
 
-This hook calls you back every frame, which is good for running effects, updating controls, etc. You receive the state (same as useThree) and a clock delta. If you supply a render priority greater than zero it will switch off automatic rendering entirely, you can then control rendering yourself. If you have multiple frames with a render priority then they are ordered highest priority last, similar to the web's z-index. Frames are managed, three-fiber will remove them automatically when the component that holds them is unmounted.
+Allows you to execute code on every frame rendered, like running effects, updating controls, and so on. You receive the state (same as useThree) and a clock delta. In addition to the callback function itself, you may pass a numerical `renderPriority` value; callbacks will be executed in order of ascending priority values (lowest first, highest last.) 
```

Comment:
It would be useful specifying wether, by default, the callback is called before or after render

Reply:
Happy to add this. I've been assuming that these (callbacks with priority 0) are executed before the scene is rendered. Can @drcmda confirm this?

Reply:
yes, they are. i can confirm it.

Reply:
I've given the copy a final do-over and made sure to include this.